packages/shared/src/components/notifications/Toast.tsx

```diff
@@ -115,7 +107,17 @@ const Toast = ({
           aria-label="Dismiss toast notification"
         />
         {autoDismissNotifications && (
-          <Progress style={{ width: `${progress}%` }} />
+          <div className="flex absolute -bottom-2 left-0 justify-center w-full ease-in-out">
+            <div
+              className={classNames(
+                'h-1 bg-theme-status-cabbage rounded-full transition-all ease-linear',
+                isAnimating ? 'w-0' : 'w-full',
+              )}
+              style={{
+                transitionDuration: `${toast.timer}ms`,
+              }}
```

Comment:
Just wondering if this is not available as a tailwind class, where we could limit the timing maybe?